modules/apps/on-demand-admin/on-demand-admin-impl/src/main/java/com/liferay/on/demand/admin/internal/ticket/generator/OnDemandAdminTicketGeneratorImpl.java

```diff
@@ -133,6 +140,34 @@ private User _addOnDemandAdminUser(Company company, User requestorUser)
 		}
 	}
 
+	private void _checkRequestAdministratorAccessPermission(
```

Comment:
IHey @RenanUmV, I expected to see the `OnDemandAdminHelper` deletion in this commit. Where is it?